account_offbalance_sponsorship/models/res_config.py

```diff
@@ -0,0 +1,48 @@
+##############################################################################
+#
+#    Copyright (C) 2014-today Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: David Wulliamoz <dwulliamoz@compassion.ch>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+
+from odoo import api, fields, models
+
+
+class OffBalanceAccountConfigSettings(models.TransientModel):
+    """
+    Add the possibility to define an off balance asset account
+    and off balance receivable account
+    """
+
+    _inherit = "res.config.settings"
+
+    account_offbalance_receivable = fields.Many2one("account.account", readonly=False)
+    account_offbalance_asset = fields.Many2one("account.account", readonly=False)
```

Comment:
Could we simply add the `company_dependent=True` property and remove the `get_values` and `set_values` overrides? To be Tested
```suggestion
    account_offbalance_receivable = fields.Many2one("account.account", company_dependent=True)
    account_offbalance_asset = fields.Many2one("account.account", company_dependent=True)
```